optax/_src/alias.py

```diff
@@ -126,16 +126,25 @@ def radam(learning_rate: ScalarOrSchedule,
 
 def rmsprop(learning_rate: ScalarOrSchedule,
             decay: float = 0.9,
+            momentum: float = 0.,
```

Comment:
Lets have `None` as default,
and check `if momentum is not None`

Reply:
change made

Reply:
Could you also move `initial_scale` right after `eps` (so that all the arguments of `scale_by_rms`  stay next to each other), and then move `momentum` at the end just before `nesterov`?

Reply:
done